content/intermediate/operations/understanding-changesets/index.md

```diff
@@ -104,12 +206,71 @@ You will now modify the value for a property, `BucketName`, that requires a [rep
 
 Let’s get started!
 
-1. In the CloudFormation console, select the `changesets-workshop` stack, and from **Stack actions**, choose **Create change set for current stack**.
-2. From **Prepare template**, choose **Use current template** and choose **Next**.
-3. Change the value for `BucketName` parameter by specifying a new unique bucket [name](https://docs.aws.amazon.com/AmazonS3/latest/userguide/bucketnamingrules.html), and follow the rest of the process as before to finish creating the change set.
+   :::::tabs{variant="container"}
+	::::tab{id="cloud9" label="Cloud9"}
+  From Terminal run the following command to **Create change set**, Change the value for `BucketName` parameter by specifying a new unique bucket [name](https://docs.aws.amazon.com/AmazonS3/latest/userguide/bucketnamingrules.html), and follow the rest of the process as before to finish creating the change set.
+  :::code{language=shell showLineNumbers=false showCopyAction=true}
+  aws cloudformation create-change-set \
+      --stack-name changesets-workshop \
+      --change-set-name replace-change-set
+      --template-body file://bucket.yaml \
+      --parameters ParameterKey=BucketName,ParameterValue='NEW_UNIQUE_BUCKET_NAME-HERE'
+  :::
+  
+  Here’s what the **JSON changes** for this change set should look like:
```

Comment:
suggest moving this entire "navigate to the console and look at the changeset JSON" to outside the tabbed instructions - it's the same either way